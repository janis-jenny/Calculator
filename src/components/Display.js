import PropTypes from 'prop-types';

function Display({ result }) {
  return (
    <div>
      <p>{ result }</p>
    </div>
  );
}

Display.propTypes = {
  result: PropTypes.string,
};

Display.defaultProps = { result: '0' };

export default Display;
