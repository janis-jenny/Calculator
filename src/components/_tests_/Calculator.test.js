import renderer from 'react-test-renderer';
import Display from '../Display';

describe('Calculator', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <>
        <Display />
      </>,
    );
    expect(tree).toMatchSnapshot();
  });
});
