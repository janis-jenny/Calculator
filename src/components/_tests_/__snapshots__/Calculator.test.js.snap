// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculator renders correctly 1`] = `
<div
  className="display"
>
  0
</div>
`;
